src/main/java/seedu/duke/storage/StorageManager.java

```diff
@@ -40,35 +98,44 @@ public static void saveNotebook(Notebook notebook) throws SystemException {
         }
     }
 
-
     /**
-     * Saves all the Events in the Timetable to the storage file.
+     * Clears the content in the original file storing all the note details.
+     * Replaces it with the new note content details.
      *
-     * @param timetable The Timetable containing all the events to be saved.
+     * @param notebook notebook that stores all the notes to be saved
+     * @throws IOException thrown when unable to write to the file
      */
-    private void saveTimetable(Timetable timetable){
-
+    public static void saveAllNoteDetails(Notebook notebook) throws IOException {
+        String path = FOLDER_DIR + NOTEBOOK_FILE_PATH;
+        FileWriter fw = new FileWriter(path);
+        fw.write("");
+        fw.close();
+
+        for (Note note: notebook.getNotes()) {
+            saveNoteDetails(note);
+        }
```

Comment:
Currently we have 2 arraylist, 1 for all the notes and 1 for archived notes. Need another loop for the getArchivedNotes()

@R-Ramana Here is the saving part.